auth_backends/backends.py

```diff
@@ -0,0 +1,114 @@
+"""Django authentication backends.
+
+For more information visit https://docs.djangoproject.com/en/dev/topics/auth/customizing/.
+"""
+import json
+
+from django.conf import settings
+import django.dispatch
+from social.backends.open_id import OpenIdConnectAuth
+
+
+# pylint: disable=abstract-method
+class EdXOpenIdConnect(OpenIdConnectAuth):
+    name = 'edx-oidc'
+
+    ACCESS_TOKEN_METHOD = 'POST'
+    REDIRECT_STATE = False
+    ID_KEY = 'preferred_username'
+
+    DEFAULT_SCOPE = ['openid', 'profile', 'email', 'permissions'] + getattr(settings, 'COURSE_PERMISSIONS_SCOPE', ['course_staff'])
```

Comment:
If I'm understanding this correctly, these are additional bits being requested by the backend from the edX Oauth2 provider. I think we want to keep asking for `openid`, `profile`, and `email`, but we don't want to hardcode requests for the `permissions` and `course_staff` bits.